packages/cli/src/lib/commands/sequence.ts

```diff
@@ -69,6 +69,30 @@ export const sequence: CommandDefinition = (program) => {
         .description("create archived file (package) with sequence for later use")
         .action((path, { stdout, output }) => packAction(path, { stdout, output }));
 
+    sequenceCmd
```

Comment:
At the end of this file is already prepared command for prune in developer mode- this should be moved down and merged with that one (remove developer flag)  to keep proper command order.